### DISCUSSION TITLE
Cannot be correctly navigated to the right layout，when using the embedded route

### DISCUSSION THREAD

**Discussion by shellingfordly** [posva/unplugin-vue-router#268] [Questions and Help] [answered]
The home page should use the home layout.  The ID page has the same problem.
The About page is no problem.

<img width="140" alt="image" src="https://github.com/posva/unplugin-vue-router/assets/39196952/27bc7c3f-dd1a-4823-8abe-21da0028d5b4">
<img width="137" alt="image" src="https://github.com/posva/unplugin-vue-router/assets/39196952/1ee9de82-56db-48b5-a314-896de3918bbd">
<img width="130" alt="image" src="https://github.com/posva/unplugin-vue-router/assets/39196952/36d816ad-db91-499c-b035-6598df35d08b">
<img width="137" alt="image" src="https://github.com/posva/unplugin-vue-router/assets/39196952/ac09357d-3981-4fea-a220-264d33f6a472">

Expected behavior

the home page uses the home layout.

Reproduction

https://stackblitz.com/~/github.com/shellingfordly/auto_router_test

**Comment by shellingfordly** [comment]
when using vite-plugin-pages, can't this problem.

**Comment by posva** [comment, marked as answer]
This seems similar to https://github.com/JohnCampionJr/vite-plugin-vue-layouts/issues/134 and it looks like you are using an outdated version of the vite-plugin-vue-layouts

**Reply by shellingfordly** [reply]
Thank you, this issue does not occur after upgrading to 0.11.0 of the vite-plugin-vue-layouts
